src/System.Runtime/tests/System/StringTests.netcoreapp.cs

```diff
@@ -1180,5 +1179,494 @@ public unsafe static bool IsSimpleActiveCodePage
                 }
             }
         }
+
+        [Theory]
+        [InlineData("  Hello  ", new char[] { ' ' }, "Hello")]
+        [InlineData(".  Hello  ..", new char[] { '.' }, "  Hello  ")]
+        [InlineData(".  Hello  ..", new char[] { '.', ' ' }, "Hello")]
+        [InlineData("123abcHello123abc", new char[] { '1', '2', '3', 'a', 'b', 'c' }, "Hello")]
+        [InlineData("  Hello  ", null, "Hello")]
+        [InlineData("  Hello  ", new char[0], "Hello")]
+        [InlineData("      \t      ", null, "")]
+        [InlineData("", null, "")]
+        [InlineData("      ", new char[] { ' ' }, "")]
+        [InlineData("aaaaa", new char[] { 'a' }, "")]
+        [InlineData("abaabaa", new char[] { 'b', 'a' }, "")]
+        public static void Trim_Memory(string s, char[] trimChars, string expected)
```

Comment:
Previous commits had this here: `src\Common\tests\Tests\System\StringTests.cs`.
But per @ahsonkhan advice, these are NetCore3, specific and belong in this file instead.